Create sales-development-representative JD.md (#4886)
FlowFuse/website: src/handbook/peopleops/job-descriptions/sales-development-representative.md

---
navTitle: Sales Development Representative (SDR)
navGroup: Job Descriptions
---

# Sales Development Representative

## Job Description

> We are hiring our **first-ever SDR** at FlowFuse — a blank-slate, build-it-yourself opportunity for someone who is energised by that kind of challenge.

Reviewer: ```suggestion
We are hiring our first SDR at FlowFuse — a blank-slate, build-it-yourself opportunity for someone who is energised by that kind of challenge.
```

There is no existing SDR playbook. No inherited cadences. No handed-down process to maintain. What exists is a growing pipeline of inbound interest, a lean and high-performing sales team, and the expectation that you will figure out — fast — what works and do more of it.

> Your mandate is broad by design: work leads. Nurture them. Find the approaches that move conversations forward and double down on those. Drop what doesn't. Whether that is relentless inbound follow-up, creative outbound sequences, personalised video messages, or something we haven't thought of yet — the right answer is whatever produces results.

Reviewer 2: ```suggestion
Your mandate is broad by design: work leads. Nurture them. Find the approaches that move conversations forward and double down on those. Drop what doesn't. Move fast. Learn faster. Whether that is relentless inbound follow-up, creative outbound sequences, personalised video messages, or something we haven't thought of yet — the right answer is whatever produces results.
```

**What you will own:**

* Work every qualified lead — inbound and, in time, outbound — toward a clear next step
* Run multi-touch follow-up sequences and refine them based on actual conversion data
* Build and maintain the SDR and BDR playbook from scratch, iterating rapidly on what works
* Increase meeting generation and reduce stalled leads
* Collaborate closely with Account Executives to ensure smooth handoff into discovery
* Feed structured insight back to marketing on lead quality, objections, and messaging gaps
* Track and report on core metrics: speed to first touch, MQL-to-meeting rate, follow-up velocity

## Role Evolution

We are hiring for inbound MQL follow-up first because that is the clearest constraint right now — but this role was never meant to stay there. As the inbound motion becomes repeatable, you will expand into outbound prospecting: identifying target accounts, testing cold sequences, and generating net-new pipeline. The SDR function you build will grow into a full SDR/BDR capability. This is a ground-floor opportunity to define what that looks like at FlowFuse.

> 

Reviewer 2: I'd add a short "We We Are" section about the team.

Example:

We work hard and we ship things. Results matter more than hours, and "good enough to learn from" beats "perfect but never shipped." Nobody is going to hand you a task list here — you're expected to find the work, jump in, and figure it out. If something doesn't work, say so, change course, and move on. No drama, no ego. Just a small team building something real in a space that most software companies ignore, having a lot of fun doing it. If you've ever thrived in an environment where everyone just figures it out, you'll fit right in.

## Skills

* **Builder mentality**: Excited to work without a blueprint. You see an undefined process as an opportunity, not a gap.
* **Bias for action**: You send the first email before the playbook is finished. You learn by doing, not by planning.
* **Inbound conversion instincts**: Speed, persistence, and follow-up quality over raw volume.
* **Iterative discipline**: You track what you test, draw conclusions quickly, and change course without drama.
* **Communication**: Strong written and verbal skills. You can engage both a plant engineer and a CTO without switching decks.
* **Technical curiosity**: Comfortable learning and talking about Node-RED, industrial automation, IoT, and edge deployments.
* **Process thinking**: You spot where leads stall, where CRM data lies, and how to fix both.

> * **Familiarity with AI tools** to accelerate prospecting, personalisation, and cadence workflows is a plus.

Reviewer 2: I'd move the AI tools up in this list so it's not an after thought and reword it so it's not optional.

`AI fluency: Comfortable using AI tools to accelerate your work — or actively developing that skill. This isn't optional, it's how we operate.`

* Start systematically testing and iterating: messaging, cadence length, channel mix, personalisation approaches
* Identify the top two or three places where MQL-to-meeting conversion breaks down and propose fixes
* Begin tracking and sharing conversion metrics weekly

**Week 9–13**
* Operate the inbound SDR motion independently with clear daily and weekly discipline
* Publish a first version of the SDR playbook — even if it is a draft — and treat it as a living document
* Provide structured feedback to marketing on lead quality and messaging gaps
* Begin scoping the outbound motion: target account criteria, sequencing approach, first tests

## Hiring Plan

* **Initial Screening**: Resume and cover letter review for communication quality, early-stage GTM experience, and evidence of a builder or tester mindset
* **First Interview**: Phone or video conversation on execution style, iteration approach, and mindset in ambiguous environments
* **Second Interview**: In-depth discussion with the hiring manager and relevant team members on process thinking, collaboration with marketing and AEs, and what "building a playbook" means to you

> * **STAR Interview (45 mins)**: Behavioral interview focused on values alignment — ownership, learning, and consistency — conducted by the CEO

Reviewer 2: ```suggestion
* **STAR Interview (45 mins)**: Behavioral interview focused on values alignment — ownership, iteration, learning from failure, and results over perfection — conducted by the CEO
```

---
navTitle: Sales Development Representative (SDR)
navGroup: Job Descriptions
---

# Sales Development Representative

## Job Description

> We are hiring our first SDR at FlowFuse — a blank-slate, build-it-yourself opportunity for someone who is energised by that kind of challenge.

Author: ```suggestion
We are hiring our first SDR at FlowFuse. This is an early-stage role where you will help shaping how we handle inbound and outbound sales development. 
```

> There is no existing SDR playbook. No inherited cadences. No handed-down process to maintain. What exists is a growing pipeline of inbound interest, a lean and high-performing sales team, and the expectation that you will figure out — fast — what works and do more of it.

Author: ```suggestion
We don't have an established SDR process or playbook yet. Part of this role is helping us figure out what works and making it repeatable. We already have a steady flow of inbound interest and a small sales team. We need someone who can follow up consistently, learn quickly, and improve the process over time.
```

> Your mandate is broad by design: work leads. Nurture them. Find the approaches that move conversations forward and double down on those. Drop what doesn't. Move fast. Learn faster. Whether that is relentless inbound follow-up, creative outbound sequences, personalised video messages, or something we haven't thought of yet — the right answer is whatever produces results.

Author: ```suggestion
Your job is simple in principle: work leads and move conversations forward. Nurture them. Find the approaches that move conversations forward and double down on those. Drop what doesn't. Move fast. Learn faster. That could mean fast inbound follow-up, testing outbound outreach, trying different messaging styles, or experimenting with new approaches.
```

> [Results](/handbook/company/values/#-results) and [Iterative Improvement](/handbook/company/values/#-iterative-improvement) are two of FlowFuse's core values. This role is where both are most visible. We expect you to ship your first outreach on day two, measure it by day five, and iterate from there. You will own the SDR and BDR playbook — not inherit it. That means writing it, testing it, revising it, and making it repeatable.

Author: ```suggestion
[Results](/handbook/company/values/#-results) and [Iterative Improvement](/handbook/company/values/#-iterative-improvement) are two of FlowFuse's core values. This role is where both are most visible. We expect you to start reaching out to leads early, learn from the responses, and improve quickly. You will own the SDR and BDR playbook, not inherit it. That means writing it, testing it, revising it, and making it repeatable.
```

**What you will own:**

> * Work every qualified lead — inbound and, in time, outbound — toward a clear next step

Reviewer: ```suggestion
* Work every qualified lead — inbound and outbound — toward a clear next step
```

* Run multi-touch follow-up sequences and refine them based on actual conversion data

> * Build and maintain the SDR and BDR playbook from scratch, iterating rapidly on what works

Reviewer: Will SDRs handle all MQLs, inbound too?

```suggestion
* Build and maintain the in- and outbound playbooks, iterating rapidly on what works
   * Communicate with team members about how to use sequences, which are available.
```

* Increase meeting generation and reduce stalled leads
* Collaborate closely with Account Executives to ensure smooth handoff into discovery
* Feed structured insight back to marketing on lead quality, objections, and messaging gaps
* Track and report on core metrics: speed to first touch, MQL-to-meeting rate, follow-up velocity

## Role Evolution

We are hiring for inbound MQL follow-up first because that is the clearest constraint right now — but this role was never meant to stay there. As the inbound motion becomes repeatable, you will expand into outbound prospecting: identifying target accounts, testing cold sequences, and generating net-new pipeline. The SDR function you build will grow into a full SDR/BDR capability. This is a ground-floor opportunity to define what that looks like at FlowFuse.

## Who We Are

> We work hard and we ship things. Results matter more than hours, and learning fast beats waiting for perfect. Nobody is going to hand you a task list here — you're expected to find the work, jump in, and figure it out. If something doesn't work, say so, change course, and move on. No drama, no ego. Just a small team building something real in a space that most software companies ignore, having a lot of fun doing it. If you've ever thrived in an environment where everyone just figures it out, you'll fit right in.

Author: ```suggestion
We move quickly and focus on getting useful work done. We care more about outcomes than appearances, and we prefer testing, iterating and learning over over-planning. This is a role for someone comfortable working independently and taking initiative without needing constant direction. If something isn't working, we expect people to speak up and adjust quickly. Just a small team building something real in a space that most software companies ignore, having a lot of fun doing it. If you've ever thrived in an environment where everyone just figures it out, you'll fit right in.
```

## Skills

* **Builder mentality**: Excited to work without a blueprint. You see an undefined process as an opportunity, not a gap.
* **AI fluency:** Comfortable using AI tools to accelerate your work — or actively developing that skill. This isn't optional, it's how we operate.

> * **Bias for action**: You send the first email before the playbook is finished. You learn by doing, not by planning.

Author: ```suggestion
* **Comfortable taking initiative and trying things quickly**: You send the first email before the playbook is finished. You learn by doing, not by planning.
```

> * **Inbound conversion instincts**: Speed, persistence, and follow-up quality over raw volume.

Author: ```suggestion
* **Strong follow-up habits**: Speed, persistence, and follow-up quality over raw volume.
```

* **Iterative discipline**: You track what you test, draw conclusions quickly, and change course without drama.
* **Communication**: Strong written and verbal skills. You can engage both a plant engineer and a CTO without switching decks.
* **Technical curiosity**: Comfortable learning and talking about Node-RED, industrial automation, IoT, and edge deployments.
* **Process thinking**: You spot where leads stall, where CRM data lies, and how to fix both.

## 90-Day Plan

**Week 1–4**

> * Immerse yourself in FlowFuse's product, market, and current GTM motion — specifically how inbound demand currently does (and does not) become meetings

Author: ```suggestion
* Learn how FlowFuse's product, customers, and current sales process work, specifically how inbound demand currently does (and does not) become meetings
```

* Learn the MQL definition, current routing logic, and any existing follow-up patterns

> * By week 2: send your first outreach sequences — no waiting for a perfect playbook

Author: ```suggestion
* By week 2: send your first outreach sequences. Start testing outreach early rather than over-planning
```

* Meet with sales, marketing, and customer success to understand the full lead lifecycle
* Begin logging observations on where leads stall and why

**Week 5–8**
* Own daily inbound MQL follow-up independently against agreed SLAs
* Start systematically testing and iterating: messaging, cadence length, channel mix, personalisation approaches
* Identify the top two or three places where MQL-to-meeting conversion breaks down and propose fixes
* Begin tracking and sharing conversion metrics weekly

**Week 9–13**
* Operate the inbound SDR motion independently with clear daily and weekly discipline
* Publish a first version of the SDR playbook — even if it is a draft — and treat it as a living document
* Provide structured feedback to marketing on lead quality and messaging gaps

> * Begin scoping the outbound motion: target account criteria, sequencing approach, first tests

Author: ```suggestion
* Start exploring what an outbound process could look like: target account criteria, sequencing approach, first tests
```

## Hiring Plan

> * **Initial Screening**: Resume and cover letter review for communication quality, early-stage GTM experience, and evidence of a builder or tester mindset

Author: ```suggestion
* **Initial Screening**: Resume and cover letter review for communication quality, early-stage GTM experience, and evidence of initiative and willingness to experiment
```

* **First Interview**: Phone or video conversation on execution style, iteration approach, and mindset in ambiguous environments
* **Second Interview**: In-depth discussion with the hiring manager and relevant team members on process thinking, collaboration with marketing and AEs, and what "building a playbook" means to you
* **STAR Interview (45 mins)**: Behavioral interview focused on values alignment — ownership, iteration, learning from failure, and results over perfection — conducted by the CEO
* **Assessment/Presentation**: Short work sample or live exercise (see below)
* **Final Interview**: Conversation with key stakeholders
* **Offer**

## Assessment

Prepare a 10-minute roleplay or presentation for a hypothetical inbound lead who has requested contact after engaging with FlowFuse content. Assume the prospect is interested in managing Node-RED instances across distributed industrial environments but has not yet committed to a meeting.

> Your goal: identify the best follow-up approach for this lead, ask the right qualifying questions, handle a first layer of objections, and move toward a booked discovery meeting.

Author: ```suggestion
Your goal: identify the best follow-up approach for this lead, ask the right qualifying questions, respond to initial objections or concerns, and move toward a booked discovery meeting.
```
